classes/student.rb

```diff
@@ -1,7 +1,9 @@
 require_relative 'person'
 
 class Student < Person
-  def initialize(classroom, age, name = 'unknown', parent_permission: true)
+  attr_accessor :classroom
+
+  def initialize(age, name = 'unknown', classroom = 'unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
   end
```

Comment:
- [ ] Kindly, make sure that you implement a method that serves as the `belongs_to` relationship between Classroom and Student as student belongs to classroom. have a look on the code below:
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```